_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
+- Current portals and accounts
+- OneGC
+- API Store
+- Pan-Canadian Trust Framework
+- Some benefits integrated with 3rd parties
+
 ## Businesses
 
+- Able to find and integrate with 50% of ESDC services
+
 ## Non-profits
```

Comment:
What definition for non-profits are we including here: Advocacy groups, think-tanks?  or groups that community-serving like aid and development?

Reply:
Some ideas in this section may be towards:
-	Changes in social fabric due to pervasiveness of technology changes the way non-profits operate (aid and development)
-	Education, health, skill training may be ubiquitous (government data becomes a way to help penetrate the impact much faster and in an effective way)

Reply:
I agree with the training point especially, though we were asked to not focus on how our services will evolve, but rather how we can leverage IT to better deliver services. 

If we consult with business on some of their plans for future services, perhaps we could offer how IT could help them accomplish them ...